packages/next-codemod/bin/next-codemod.ts

```diff
@@ -46,7 +46,11 @@ program
   .description(
     'Upgrade Next.js apps to desired versions with a single command.'
   )
-  .usage('[options]')
+
+  .argument(
```

Comment:
Why aren't we using a default of `latest`?

Reply:
I think the idea was to default to the prompt instead which does default to `latest`.

Though I think it makes more sense to remove the prompt and default to `latest` while using hard-coded choices.

Reply:
Went one stop further: We default now to the dist tag that the current version is likely published as: https://github.com/vercel/next.js/pull/70638